app/components/tabs.js

```diff
@@ -1,21 +1,36 @@
 var tabs = document.getElementById('tabs');
 var tab1 = document.getElementById('tab1');
 var tab2 = document.getElementById('tab2');
+var tab3 = document.getElementById('tab3');
 var tab1Title =  document.getElementById('tab1-title');
 var tab2Title =  document.getElementById('tab2-title');
+var tab3Title =  document.getElementById('tab3-title');
 
 function toggleTabs(tabId) {
 	if (tabId < 2) {
```

Comment:
condicao nao muito intuitiva, modifique para ficar de forma mais legivel e entendivel para quem for ler depois.